akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -175,12 +176,13 @@ private[projection] case class MSSQLServerDialect(schema: Option[String], tableN
 
   def this(tableName: String) = this(None, tableName)
 
-  private val table = schema.map(s => s"$s.$tableName").getOrElse(tableName)
+  private val table = schema.map(s => s""""$s"."$tableName"""").getOrElse(s""""$tableName"""")
+
   override val createTableStatements =
     immutable.Seq(s"""
       IF  NOT EXISTS (SELECT 1 FROM sys.objects WHERE object_id = OBJECT_ID(N'$table') AND type in (N'U'))
       begin
-      create table "$table" (
+      create table $table (
```

Comment:
This is where it fails in the `OffsetSchemaSpec`. 

`val table` has not the quotes. That's fine for the `create table` part, but `IF NOT EXISTS` part is now getting the schema.table wrapped with double-quotes. 

We can add the quotes to `create-table-mssql.sql` script to align the two.

Reply:
Done